scripts/scil_screenshot_bundle.py

```diff
@@ -126,11 +168,16 @@ def main():
                 parser.error('{0} is not a valid RGB value'.format(val))
 
     # Get the relevant slices from the template
-    target_template_img = nib.load(args.target_template)
+    if args.target_template:
+        mni_space_img = nib.load(args.target_template)
+        affine = nib.load(args.target_template).affine
+    else:
+        mni_space_img = nib.load(args.in_anat)
+        affine = nib.load(args.in_anat).affine
 
-    x_slice = int(target_template_img.shape[0] / 2)
-    y_slice = int(target_template_img.shape[1] / 2)
-    z_slice = int(target_template_img.shape[2] / 2)
+    x_slice = int(mni_space_img.shape[0] / 2)
+    y_slice = int(mni_space_img.shape[1] / 2)
+    z_slice = int(mni_space_img.shape[2] / 2)
     slices_choice = (x_slice, y_slice, z_slice)
 
```

Comment:
Still a bit long. Could we add something like "Starting..."?, so we know it didn't bug?